### DISCUSSION TITLE
Kong Manager Network Error

### DISCUSSION THREAD

**Discussion by aureuslim** [Kong/kong#12955] [Help]
I have 2 Kong setup with Kong Manager enabled, 1 Kong OSS in Docker, and 1 Kong Enterprise Free Version in Ubuntu.

Both my Kong Manager works when using HTTP://localhost.com, however both has the same issue of having Network Error under the Workspaces overview.

![image](https://github.com/Kong/kong/assets/49929225/2866254b-ce89-41c3-9665-c28628fcfb75)

The configurations that I have edited in kong.conf are as follows:
admin_listen = 0.0.0.0:8001 reuseport backlog=16384, 0.0.0.0:8444 http2 ssl reuseport backlog=16384
admin_gui_listen = 0.0.0.0:8002, 0.0.0.0:8445 ssl
admin_gui_url = http://localhost:8002/
admin_gui_path = /
admin_gui_api_url = http://localhost:8002/

This is the error displayed on Google Chrome Console:

GET http://localhost:8002//default/kong net::ERR_CONNECTION_REFUSED
GET http://localhost:8002//workspaces?size=15&sort_by=created_at&counter=true net::ERR_CONNECTION_REFUSED

Can anyone please let me know if I missed out on any other important configurations or did I edit any of these configurations wrongly?

I can provide more details as to what I have done in the setup, or whatever necessary.

**Comment by chronolaw** [comment]
@sumimakito , could you take a look at this?

**Comment by sumimakito** [comment]
Hello @aureuslim,

`admin_gui_api_url` sets the base URL for Admin API requests, which would be `http://localhost:8001` according to `admin_listen = 0.0.0.0:8001` in your case. 

Meanwhile, we recommend omitting the trailing slash while setting URLs such as `admin_gui_api_url` and `admin_gui_url`.

You may try to update your configuration with the following changes:

```diff
- admin_gui_api_url = http://localhost:8002/
+ admin_gui_api_url = http://localhost:8001
- admin_gui_url = http://localhost:8002/
+ admin_gui_url = http://localhost:8002
```

I hope this will be helpful. Please don't hesitate to comment if the issue continues.
